pre_award/db/__init__.py

```diff
@@ -10,8 +14,24 @@
     "pk": "pk_%(table_name)s",
 }
 
+
+class FundingType(enum.Enum):
+    COMPETITIVE = "COMPETITIVE"
+    UNCOMPETED = "UNCOMPETED"
+    EOI = "EOI"
```

Comment:
I think, as we start to add more types and have more custom defintions, we probably want a types package/module (internal to this repo)

But for now this is fine